ThePhasesOfCraftship/2_best_practice_first/strategicDesignPart1/exercises/1_RefactoringTo4Tiers/begin/src/dto/StudentRequestDto.ts

```diff
@@ -0,0 +1,40 @@
+import { isUUID } from "../utils"
+import { ErrorExceptionType } from "../constants"
+
+export type StudentRequestDto = {
```

Comment:
I think this DTO is too generic, I don't like this "one dto for all" approach. It would be more expressive to create separate ones for each operation.